bash/ubuntu-run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# If set to true, ddtrace-run will be enabled for dbt-server.
+readonly dbt_server_enable_ddtrace="${DBT_SERVER_ENABLE_DDTRACE-false}"
+# How many dbt server worker processes.
+readonly dbt_server_worker="${DBT_SERVER_WORKER_NUM-3}"
+# Dbt server listening port.
+readonly dbt_server_port="${DBT_SERVER_WORKER_PORT-8585}"
+# Max # of requests that dbt server is allowed.
+readonly dbt_server_max_requests="${DBT_SERVER_MAX_REQUESTS-5}"
+
+gunicorn="gunicorn"
+if [ "${dbt_server_enable_ddtrace}" = "true" ]; then
+    dd_trace="ddtrace-run gunicorn"
+fi
+
+sh bash/start-celery.sh
```

Comment:
have we considered using [supervisor](http://supervisord.org) to manage these various processes? it has the benefit of restarting them if they stop.
alternatively, we'll probably need to create a k8s health check that can assert that **all** processes (redis, celery, uvicorn, etc) are always running.
if we don't do this, we risk being in situations where only dbt-server is running and maybe redis or celery have failed for whatever reason

Reply:
Totally agree supervisord should be used(I left one TODO in celery.sh as ideally we should use process manager). But because I want IDE team to test ASAP I will push back onboarding supervisord a little bit, but I agree that's the ideal one. Redis I use init.d to manage but seems that supervisord is better? I need to get familiar with supervisord.

For health check I guess there might be couple of solutions
- Assert all processes are alive.
- Dbt server have a heartbeat endpoint to send a dumb task to celery and redis. I believe that's more ideal but let me take a look how k8s conducts health check first.

Reply:
cool, thanks, just wanted to make sure it was on your radar. glad to hear that it is!

Reply:
I took some time to investigate couple of options.

First question: serviced(init.d) vs supervisord: 
- Definitely supervisord is more powerful but serviced is preinstalled in our docker and much easier to setup. Both celery and redis supports serviced while we need some efforts for supervisord.

Second question: how to do health check? I learned k8s will use livenessProbe to detect pod health. Two options:
- Send a http request to dbt server. Dbt server checks if redis is alive and celery worker is alive. Celery provide inspection [API](https://docs.celeryq.dev/en/latest/reference/celery.app.control.html#celery.app.control.Control.inspect) to check all celery status, including broker redis. 
- Invoke a terminal command. Possible to check "service redis-server status" + "service celeryd status" + "dbt server status" but might be little bit tedious.

I will address the health check in a following PR.